src/OrbitQt/CaptureOptionsDialog.cpp

```diff
@@ -42,14 +55,25 @@ CaptureOptionsDialog::CaptureOptionsDialog(QWidget* parent)
     // TODO(b/198748597): Don't hide samplingCheckBox once disabling sampling completely is exposed.
     ui_->samplingCheckBox->hide();
     ui_->unwindingMethodWidget->hide();
+    ui_->maxCopyRawStackSizeLineEdit->hide();
     ui_->schedulerCheckBox->hide();
     ui_->gpuSubmissionsCheckBox->hide();
     ui_->introspectionCheckBox->hide();
   }
 
   ui_->localMarkerDepthLineEdit->setValidator(&uint64_validator_);
-  ui_->memorySamplingPeriodMsLineEdit->setValidator(new UInt64Validator(1));
+  ui_->memorySamplingPeriodMsLineEdit->setValidator(new UInt64Validator(
+      1, std::numeric_limits<uint64_t>::max(), ui_->memorySamplingPeriodMsLineEdit));
   ui_->memoryWarningThresholdKbLineEdit->setValidator(&uint64_validator_);
+  ui_->maxCopyRawStackSizeLineEdit->setValidator(
+      new UInt64Validator(0, kMaxCopyRawStackSizeMaxValue, ui_->maxCopyRawStackSizeLineEdit));
+
+  ui_->maxCopyRawStackSizeLabel->setToolTip(QString::fromStdString(
+      "To improve performance, we don't require to preserve the frame pointer register on leaf\n"
+      "functions (-momit-leaf-frame-pointer). In order to fix callstacks from samples falling\n"
```

Comment:
Should we expand on "fix"? E.g., "In order to recover the frame corresponding to the caller of a leaf function, ..."